cmd/convox/ps.go

```diff
@@ -44,6 +49,9 @@ func init() {
 }
 
 func cmdPs(c *cli.Context) error {
+	stdcli.NeedHelp(c)
+	stdcli.NeedArg(c, 0)
```

Comment:
The expected args logic makes sense but I think we should not display the red `ERROR` warning. It seems unnecessary with good Usage output and can be a bit confusing on its own.

```
$ convox ps web --app site-staging
ERROR: expected 0 arguments; got 3 arguments (web --app site-staging). Perhaps you meant to use a subcommand or option?
convox ps: list an app's processes
```